index.html

```diff
@@ -0,0 +1,134 @@
+                <nav>
+                <a href="#"><img src="images/F1_logo.png" alt="summit-logo" class="logo"></a>
+                </nav>
+            </div>
+         
+
+        <nav class="menu" id="menu-pop">
+            <img src="images/Icon-close.svg" alt="" class="close-button">
+            <ul class="nav-content">
+                <li><a href="about.html">About</a></li>
+                <li><a href="#program" id="nav-grey">Program</a></li>
+                <li><a href="#join" id="nav-grey">Join</a></li>
+                <li><a href="#sponsor" id="nav-grey">Sponsor</a></li>
+                <li><a href="#news" id="nav-grey">News</a></li>
+                <li><a href="" id="cc-link">F1Campaign</a></li>
+            </ul>
+        </nav>
+
+    </header>
+
+    <section class="intro">
+        <p id="hello">"Hello!,Sharing world"</p>
+        <div class="title-intro">
+            <h1>FORMULA 1 FANS GLOBAL SUMMIT 2023</h1>
+        </div>
+        <div class="p-intro">
+            <p class="summit-intro">Welcome to the 7th global summit of Formula 1 fans.
+                Celebrate,have fun and bless someone.Venue:Vienna,Austria. </p>
+        </div>
+        <div class="date">
+            <p class="show-date">2023.9.8(FRI) ~ 9(SAT)</p>
+            <p>@Grand Prix of Vienna and more</p>
+        </div>
+    </section>
+
+    <section class="program">
+        <h2>Main program</h2>
+        <hr class="sm-line">
+
+        <div id="gr-program">
+            <div class="grid-program">
+                <img src="images/program_icon_01 1.svg" alt="lecture-icon" class="program-icon">
+                <p class="program-points">Lecture</p>
+                <p class="program-ex">Listen to the speakers from various countries 
+                    about the messages of sharing and opening.</p>
+            </div>
+
+            <div class="grid-program">
+                <img src="images/program_icon_02 1.svg" alt="exhibition-icon" class="program-icon">
+                <p class="program-points">CC Exhibition</p>
+                <p class="program-ex">Appreciate various creations applying 
+                    CC license of artists, organized from Art Center Nabi</p>
+            </div>
+
+            <div class="grid-program">
+                <img src="images/program_icon_03 1.svg" alt="forum-icon" class="program-icon">
+                <p class="program-points">Forum</p>
+                <p class="program-ex">Have the time to share your thoughts and 
+                    opinions with experts for each topic.</p>
+            </div>
+
+            <div class="grid-program">
+                <img src="images/program_icon_04 1.svg" alt="workshop-icon" class="program-icon">
+                <p class="program-points">Workshop</p>
+                <p class="program-ex">Try creating your own work using open source license 
+                    rather than just watching at it.</p>
+            </div>
```

Comment:
- [ ] Good job implementing the `hover` selector 👍, however, as the previous reviewer showed on the [screenshot image](https://github.com/MarkYegon7/Capstone-Project/pull/1#:~:text=in%20the%20Cindy-,Your%20design,-Cindy%20Design), inside of changing the background section when we hover the section, you are supposed to implement the `hover` effect on the `border section`  to stick to the original design, Kindly fix it 👍 

##### Original design
![Screenshot 2023-08-09 172346](https://github.com/MarkYegon7/Capstone-Project/assets/57408419/fe33e5cb-8aea-4b7c-b087-2f8f5c695a92)

##### Your design
![Screenshot 2023-08-09 172305](https://github.com/MarkYegon7/Capstone-Project/assets/57408419/7cc01ed7-c5f9-4cf7-b79c-d056a979f72f)

Reply:
Introduced the hover on border section